### PR TITLE
add detect YandexBrowser

### PR DESCRIPTION
**Pull request by bartigor**
Hey. Added detection of Yandex Browser

### CODE REVIEW

**Comment by toritori0318** [comment]
@bartigor Thank you for your pull request!

In the woothee project, you should share tested browser list in whole languages.
At first please send a pull request [here](https://github.com/woothee/woothee/pulls) ([example](https://github.com/woothee/woothee/pull/20/commits/d319f41d02c3f48f697fd01e73e99331dcb598ce)),
and then notify me to merge this pull request.

**Comment by bartigor** [comment]
Hello. my PR merged https://github.com/woothee/woothee/pull/43

File: lib/resty/woothee.lua [hunk 1]
```diff
@@ -38,6 +38,10 @@ local function try_browser(useragent, result)
     return true
   end
 
+  if browser.challenge_yandexbrowser(useragent, result) then
```
Comment (toritori0318) [inline lib/resty/woothee.lua]: @bartigor This code needs to be executed before `challenge_safari_chrome`.

File: lib/resty/woothee/browser.lua [hunk 2]
```diff
@@ -110,12 +110,12 @@ function _M.challenge_yandexbrowser(ua, result)
...
 
   local version = dataset.VALUE_UNKNOWN
-  local match, err = ngx.re.match(ua, [[YaBrowser/(\d+\.\d+\.\d+)]], "o")
+  local match, err = ngx.re.match(ua, [[YaBrowser/(\d+\.\d+\.\d+\.\d+)]], "o")
   if match then
     version = match[1]
   end
 
-  util.update_map(result, dataset.get('YandexBrowser'))
+  util.update_map(result, dataset.get('Yandex Browser'))
```
Comment (toritori0318) [inline lib/resty/woothee/browser.lua]: @bartigor Please specify `YaBrowser` for dataset key.

**Comment by toritori0318** [comment]
All tests passed! 🎉  
Thanks! 👍

**Comment by yuya-takeyama** [comment]
@toritori0318 @bartigor 
Hi, don't you need to update the submodule?
It still refers an old revision.
https://github.com/woothee/woothee/tree/9e1c4953bce098e7ceb7096655684d611b94720a

**Comment by toritori0318** [comment]
@yuya-takeyama  
Hi, I plan to reflect it after fixing woothee version.

**Pull request merged**